### ISSUE TITLE
문제 YOLOv5 구현

### ISSUE THREAD

**Issue by Batwan01** [ultralytics/yolov5#13340]

**Comment by UltralyticsAssistant** [comment]
👋 안녕하세요 @Batwan01, YOLOv5 🚀에 관심을 가져주셔서 감사합니다! 

**Tutorials**를 방문하여 시작해 보세요: [Tutorials](https://docs.ultralytics.com/yolov5/). 여기에는 [Custom Data Training](https://docs.ultralytics.com/yolov5/tutorials/train_custom_data/)과 같은 간단한 작업부터 [Hyperparameter Evolution](https://docs.ultralytics.com/yolov5/tutorials/hyperparameter_evolution/)과 같은 고급 개념까지 다양한 빠른 시작 가이드가 있습니다.

🐛 **버그 리포트**인 경우, 디버깅에 도움이 되는 **최소 재현 가능한 예제**를 제공해주세요.

❓ **커스텀 트레이닝 질문**인 경우, 데이터셋 이미지 예제 및 트레이닝 로그와 함께 가능한 많은 정보를 제공해 주세요. 또한 [최고의 훈련 결과를 위한 팁](https://docs.ultralytics.com/guides/model-training-tips/)을 따르고 있는지 확인해 주세요.

## 요구 사항

[**Python>=3.8.0**](https://www.python.org/)과 모든 [requirements.txt](https://github.com/ultralytics/yolov5/blob/master/requirements.txt)에 나열된 패키지가 설치된 환경이 필요합니다, 특히 [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/)가 포함되어 있습니다. 시작하려면:
```bash
git clone https://github.com/ultralytics/yolov5  # clone
cd yolov5
pip install -r requirements.txt  # install
```

## 실행 환경

YOLOv5는 다음과 같은 최신 검증된 환경에서 실행할 수 있습니다 (모든 종속성 포함 [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/), 그리고 [PyTorch](https://pytorch.org/) 미리 설치):

- **Notebooks** with free GPU: <a href="https://bit.ly/yolov5-paperspace-notebook"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"></a> <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM: [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI: [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**: [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>

## 상태

<a href="https://github.com/ultralytics/yolov5/actions/workflows/ci-testing.yml"><img src="https://github.com/ultralytics/yolov5/actions/workflows/ci-testing.yml/badge.svg" alt="YOLOv5 CI"></a>

이 배지가 초록색이면, 모든 [YOLOv5 GitHub Actions](https://github.com/ultralytics/yolov5/actions) Continuous Integration (CI) 테스트가 현재 성공적으로 진행 중임을 의미합니다. CI 테스트는 YOLOv5의 [training](https://github.com/ultralytics/yolov5/blob/master/train.py), [validation](https://github.com/ultralytics/yolov5/blob/master/val.py), [inference](https://github.com/ultralytics/yolov5/blob/master/detect.py), [export](https://github.com/ultralytics/yolov5/blob/master/export.py), 그리고 [benchmarks](https://github.com/ultralytics/yolov5/blob/master/benchmarks.py) 의 올바른 작동을 검증하며, macOS, Windows, Ubuntu에서 매 24시간 및 각 커밋마다 수행됩니다.

## 새로운 YOLOv8 🚀 소개

최신 최첨단(SOTA) 객체 탐지 모델인 [YOLOv8](https://github.com/ultralytics/ultralytics) 🚀 출시 소식을 전합니다!

YOLOv8는 빠르고, 정확하며 사용하기 쉬운 설계를 통해, 실시간 객체 탐지, 워크플로우 간소화, 프로젝트에서 새로운 정확성을 달성하기 위한 다양한 작업에 이상적인 선택입니다.

자세한 내용은 [YOLOv8 문서](https://docs.ultralytics.com/)를 참조하세요:
```bash
pip install ultralytics
```

이것은 자동화된 응답이며, Ultralytics의 엔지니어가 곧 더 도와드릴 것입니다. 😊

**Comment by pderrenger** [comment]
@Batwan01 please provide more details about the issue you're experiencing with YOLOv5, and I'll be happy to assist you further.
